modin/experimental/xgboost/test/test_xgboost.py

```diff
@@ -0,0 +1,172 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+import pytest
+
+import modin.experimental.xgboost as xgb
+import modin.pandas as pd
+import numpy as np
+import xgboost
+
+from sklearn.datasets import (
+    load_iris,
+    load_diabetes,
+    load_digits,
+    load_wine,
+    load_breast_cancer,
+)
+from sklearn.metrics import accuracy_score, precision_score
+from numpy.random import uniform, randint
+
+import multiprocessing as mp
+
+
+num_cpus = mp.cpu_count()
+
+
+@pytest.mark.parametrize(
+    "num_round",
+    [2, 4, 8, 16, 32],
+    ids=["2", "4", "8", "16", "32"],
+)
+@pytest.mark.parametrize(
+    "num_actors",
+    [1, num_cpus, None],
+    ids=["1", "num_cpus", "None"],
```

Comment:
Also, need to add the next case when `num_actors` > number of partitions of Modin DataFrame.

Reply:
Was added case num_actors=400

Reply:
I would prefer to use some relative value, to highlight that required number of actors is more that actual number of parts: `NPartitions.get() + 1`

Reply:
Was added `num_actors=NPartitions.get() + 1`